backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/descriptors/DescriptorUtils.kt

```diff
@@ -219,8 +219,10 @@ internal tailrec fun IrDeclaration.findPackage(): IrPackageFragment {
             ?: (parent as IrDeclaration).findPackage()
 }
 
-fun IrFunction.isComparisonFunction(map: Map<SimpleType, IrSimpleFunctionSymbol>): Boolean =
-        this.symbol in map.values
+fun IrDeclaration.isIrBuiltIn() = this.findPackage().packageFragmentDescriptor is IrBuiltinsPackageFragmentDescriptor
+
+fun IrFunction.isComparisonFunction(map: Map<SimpleType, IrSimpleFunctionSymbol>) =
+        this.symbol.descriptor in map.values.map {it.descriptor}
```

Comment:
What is this required for?

Reply:
It's used in evaluateOperatorCall https://github.com/JetBrains/kotlin-native/blob/b19d02885d89aeaaf8d7fd2d783eb24ed993ef1a/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt#L2107

I just removed extra return type.

Reply:
`this in map.values` is clearly different from `this.symbol.descriptor in map.values.map {it.descriptor}`.

Reply:
Yes, I understood, I commited experiments when was debugging. Fixed as I made before.